docs/adguard-vpn-browser-extension/solving-problems/logs.md

```diff
@@ -3,62 +3,75 @@ title: How to collect and send logs
 sidebar_position: 1
 ---
 
-If you encounter a problem when using AdGuard VPN Browser Extension, you can inform us about it by sending application logs.
+There are several choices for collecting AdGuard VPN Browser extension logs, but we want to draw your attention to the fact that whichever way you go, the same data will be collected. In the extension, logs are not divided into standard and debug ones. And different options for collecting logs are provided for those cases when a problem you encounter prevents you from using one or another instruction.
 
-## Collecting and sending standard logs
+## Collecting and sending logs via the *Report a bug* section
 
-By default, AdGuard VPN Browser Extension uses the Default logging level, that is, the basic collection of data about running processes of the app. To send these logs, follow the next steps:
+1.  Open AdGuard VPN Browser extension, if possible, repeat the actions that led to the error. Note the exact time when this error occurred.
+2.  Open *Settings* by clicking the hamburger menu icon (☰) → *Support* → *Report a bug*.
+3.  In the opened form, leave an automatically inserted email address or enter another one and describe the error found, including the time when this error occurred. If you can't reproduce the problem, specify as accurately as possible when it last occurred.
+4.  Make sure that there is a check mark next to *Include the diagnostic report in the message*, and tap *Submit*. This way, you will send logs along with the bug report.  
 
-1. Open AdGuard VPN Browser Extension, if possible, repeat the actions that led to the error. Note the exact time when this error occurred.
+## Collecting and sending logs via the *Export logs* button
 
-2. Open *Settings* by clicking the hamburger menu icon (☰) → *Support* → *Report a bug*.
+1.  Open AdGuard VPN Browser extension, if possible, repeat the actions that led to the error. Note the exact time when this error occurred. 
+2.  Right-click the AdGuard VPN Browser extension icon. 
+3.  In the opened menu click *Export Logs*.
+4.  As a result, an archive with logs will be saved to your device in `txt` format.
```

Comment:
just a txt file, not in the archive, maybe in future we will fix it and export the archive